File: app/src/main/java/com/shetj/diyalbume/test/TestApi.java
package com.shetj.diyalbume.test;

import io.reactivex.Observable;
import retrofit2.http.GET;
import retrofit2.http.Path;

/**
 * <b>@packageName：</b> com.shetj.diyalbume.test<br>
 * <b>@author：</b> shetj<br>
 * <b>@createTime：</b> 2017/12/20<br>
 * <b>@company：</b><br>
 * <b>@email：</b> 375105540@qq.com<br>
 * <b>@describe</b><br>
 */

public interface TestApi {

	@GET("api/data/{category}/{count}/{page}")
	Observable<String > getInfo(@Path("category") String category);
 }

File: app/src/main/java/com/shetj/diyalbume/test/TestApi.kt
package com.shetj.diyalbume.test

import io.reactivex.Observable
import retrofit2.http.GET
import retrofit2.http.Path

/**
 * **@packageName：** com.shetj.diyalbume.test<br></br>
 * **@author：** shetj<br></br>
 * **@createTime：** 2017/12/20<br></br>
 * **@company：**<br></br>
 * **@email：** 375105540@qq.com<br></br>
 * **@describe**<br></br>
 */

interface TestApi {

    @GET("api/data/{category}/{count}/{page}")
    fun getInfo(@Path("category") category: String): Observable<String>
}
